tvtk/wrapper_gen.py

```diff
@@ -1042,8 +1043,12 @@ def _write_tvtk_method(self, out, vtk_meth, sig=None):
           If None, this is computed.  If not, the passed signature
           information is used.
         """
-        if not sig:
+        if sig is None:
             sig = self.parser.get_method_signature(vtk_meth)
+        
+        # VTK 6.2: When the method signature is None, ignore the method
```

Comment:
I think that the comment is a little misleading, since the check does not know why the get_metho_signature returned None. We only know that when signature is None we ignore the method from wrapping.